workloads/etcd-perf/run_etcd_tests_fromgit.sh

```diff
@@ -42,35 +43,44 @@ cat << EOF | oc create -n my-ripsaw -f -
 apiVersion: ripsaw.cloudbulldozer.io/v1alpha1
 kind: Benchmark
 metadata:
-  name: fio-benchmark
+  name: etcd-fio
   namespace: my-ripsaw
 spec:
   elasticsearch:
-    server: $_es
-    port: $_es_port
-  clustername: $cloud_name
+    server: ${_es}
+    port: ${_es_port}
+  clustername: ${cloud_name}
   test_user: ${cloud_name}-ci
   metadata_collection: true
   metadata_sa: backpack-view
   metadata_privileged: true
```

Comment:
Metadata should now be in the form:
metadata:
  collection: true
  serviceaccount: backpack-view
  privileged: true

etc

Reply:
fixed!